### ISSUE TITLE
Add banks brand colors

### ISSUE THREAD

**Issue by ai** [jessepollak/card#284]
There is a good and open [banks BINs database](https://github.com/ramoona/banks-db) with banks names and brand colors.

I think we could do form UX better if we will show bank name and color while user input a card number (we could get bank by first 6 card numbers, BIN). Here is a demo: https://ramoona.github.io/banks-db-demo/

In fact, in Russia few big banks already use this UX: Alfabank, Tinkoff and Rocket bank. And all my friends love this behavior.

@jessepollak what do you think about it?

**Comment by VersatilityWerks** [comment]
Probably not good for prime time in America. Good idea tho. I tried a couple of my own cards with no luck.

**Comment by ai** [comment]
@VersatilityWerks sure. But it is a open source database. Feel free to add your BINs there.

**Comment by VersatilityWerks** [comment]
Yea, every DB starts somewhere. Just saying this one needs some work. I'll contribute when I get a moment.

**Comment by VersatilityWerks** [comment]
I'm sure there are a lot more, but I took a couple minutes to add SunTrust and Navy Federal. Also, the list of BINs I found for SunTrust may have only been for their Debit Cards. May need to have another set of them added for Credit Cards. Not sure on that.

**Comment by ai** [comment]
Great, thanks!

@jessepollak what do you think about this idea?

**Comment by jessepollak** [comment]
@ai I think it's a good idea, though I'm also wary of switching to this if it degrades the ability to detect card providers etc. Are you suggesting we switch entirely to this for card detection or add it as an additional layer on top of our current card detection? Also, do you know how large the dataset is to do this detection?

**Comment by VersatilityWerks** [comment]
@jessepollak - They only have a couple/few for the United States and it's a very incomplete library. Might be cool to add on top of existing detection to show specific banks when they exist, but I would NOT recommend switching completely.

**Comment by ai** [comment]
@VersatilityWerks I agree that this library don’t have all banks, but it is the most complete database of banks brand colors.

Card library right now has some different database for banks names?

**Comment by ai** [comment]
@jessepollak “card providers” is bank names (Citi Bank, Bank of America, …) or card types (Visa/MasterCard/…)?

**Comment by VersatilityWerks** [comment]
Another concern is lack of control over background color to make the font contrast. If a bank's color is navy blue, you probably can't have black text over the top. There will need to be additional considerations.

**Comment by ai** [comment]
@VersatilityWerks a11n is a good concern.

This is why banks-db suggests to use `postcss-contrast` for CSS-only or `contrast` for CSS-in-JS implementation.

```js
background: color,
color: contrast(color) === 'light' ? 'black' : 'white'
```

**Comment by VersatilityWerks** [comment]
That works? Dang. Never seen that before. Cool.

**Comment by shkupjane** [comment]
how can i use binlist in this plugin @jessepollak ?
i want get bank name. can u help me plz?
